### PR TITLE
FSE: Modern business - add site logo placeholder to header template

### PR DESCRIPTION
**Pull request by glendaviesnz**
#### Changes proposed in this Pull Request:

* Add a site logo placeholder to the FSE header template along with relevant editor and front end styles

#### Testing instructions

* Build Modern Business theme and copy to  FSE test environment, and then follow test instructions at https://github.com/Automattic/wp-calypso/pull/35152

#### Related issue(s):

https://github.com/Automattic/wp-calypso/pull/35152
https://github.com/Automattic/wp-calypso/issues/35124

### CODE REVIEW

**Comment by noahtallen** [comment]
cc @vindl, how should this integrate with the new API work?

**Comment by glendaviesnz** [comment]
> cc @vindl, how should this integrate with the new API work?

The additional image markup will just be added to the  demo site template markup instead of hardcoded in template class.

**Comment by vindl** [comment]
> cc @vindl, how should this integrate with the new API work?

What @glendaviesnz said. Let me know if you need help adding it to the demo site.

**Comment by glendaviesnz** [comment]
This has been updated now and the image header block data added to the modern-business theme api, so can be tested

File: modern-business/sass/site/header/_site-header.scss [hunk 1]
```diff
@@ -24,6 +24,14 @@
 			margin-bottom: 2rem;
 		}
 	}
+
+	.fse-site-logo {
+		margin-bottom: 0;
+        img {
```
Comment (jeraldjuice) [inline modern-business/sass/site/header/_site-header.scss]: Nit: this is happening because some of these indents are spaces vs. tabs.

Comment (glendaviesnz) [reply modern-business/sass/site/header/_site-header.scss]: sorted :-) - well that little issue is, now I just need to fix VS Code's format on save for these files so I don't need to manually add indents and get it wrong.

**Pull request merged**